docs/destroy-cluster.md

```diff
@@ -0,0 +1,70 @@
+# Destroying a Cluster
+
+When a cluster is no longer needed and needs to be torn down, Rook has a [Cleanup guide](https://rook.io/docs/rook/latest/Getting-Started/ceph-teardown/) with instructions to tear it down cleanly. While following that document is highly preferred, there is a command that will automate the cleanup of Rook after applications are removed.
+The `destroy-cluster` command destroys a Rook cluster and deletes CRs (custom resources) by force removing any finalizers that may be preventing the cleanup.
+
+## !!! Warning !!!
+**This command is not reversible**, and it will destroy your Rook cluster completely with guaranteed data loss.
+Only use this command if you are 100% sure that the cluster and its data should be destroyed.
+
+## `destroy-cluster` command
+
+To destroy a cluster, run the command:
+```bash
+$ kubectl rook-ceph -n rook-ceph destroy-cluster
+Warning: Are you sure you want to destroy the cluster in namespace rook-ceph? If absolutely certain, enter: yes-really-destroy-cluster
+```
+
+Any other response will cause the command to abort.
+
+## Example of destroying a cluster
+
+```bash
+$ kubectl rook-ceph -n rook-ceph destroy-cluster
+Warning: Are you sure you want to destroy the cluster in namespace "rook-ceph"? If absolutely certain, enter: yes-really-destroy-cluster
+yes-really-destroy-cluster
+Info: proceeding
+Info: Getting resources kind cephclusters
+Info: removing resource cephclusters: my-cluster
+Info: resource "my-cluster" is not yet deleted, applying patch to remove finalizer...
+Info: resource my-cluster was deleted
+Info: Getting resources kind cephblockpoolradosnamespaces
+Info: resource cephblockpoolradosnamespaces was not found on the cluster
+Info: Getting resources kind cephblockpools
+Info: removing resource cephblockpools: builtin-mgr
+Info: resource "builtin-mgr" is not yet deleted, applying patch to remove finalizer...
+Info: cephblockpools.ceph.rook.io "builtin-mgr" not found
+Info: Getting resources kind cephbucketnotifications
+Info: resource cephbucketnotifications was not found on the cluster
+Info: Getting resources kind cephbuckettopics
+Info: resource cephbuckettopics was not found on the cluster
+Info: Getting resources kind cephclients
+Info: resource cephclients was not found on the cluster
+Info: Getting resources kind cephcosidrivers
+Info: resource cephcosidrivers was not found on the cluster
+Info: Getting resources kind cephfilesystemmirrors
+Info: resource cephfilesystemmirrors was not found on the cluster
+Info: Getting resources kind cephfilesystems
+Info: removing resource cephfilesystems: myfs
+Info: cephfilesystems.ceph.rook.io "myfs" not found
```

Comment:
Seems like a bug that the myfs filesystem was found to remove (line 48), but then it was not found on line 49.